.github/CONTRIBUTING.md

```diff
@@ -59,26 +59,32 @@ The basic workflow for contributing is:
    ```
    git checkout -b name-of-your-branch
    ```
-4. Install the dependencies listed in `requirements.txt` and `requirements-dev.txt`.
+4. Install the [miniconda](https://docs.conda.io/en/latest/miniconda.html) to avoid any external library errors.
+
+   If using `conda` one can create a development environment with:
+   ```
+   $ conda create --name FOLIUM python=3.7 --file requirements.txt --file requirements-dev.txt
+   ```
+5. Install the dependencies listed in `requirements.txt` and `requirements-dev.txt`.
    ```
    pip install -r requirements.txt
    pip install -r requirements-dev.txt
    ```
-5. Install Firefox, download [geckodriver](https://github.com/mozilla/geckodriver/releases) 
+6. Install Firefox, download [geckodriver](https://github.com/mozilla/geckodriver/releases) 
    and put it in the PATH.
-6. Make changes to your local copy of the folium repository
-7. Make sure the tests pass:
+7. Make changes to your local copy of the folium repository
+8. Make sure the tests pass:
    * in the repository folder do `pip install -e .`  (needed for notebook tests)
```

Comment:
One last thing that is unrelated but now is a good time to fix it. Please you change the `pip` install command to:

```
pip install -e . --no-deps
```

Reply:
Fixed